google-cloud-spanner/src/main/java/com/google/cloud/spanner/v1/SpannerClient.java

```diff
@@ -184,6 +184,9 @@ public static final SpannerClient create(SpannerStub stub) {
   protected SpannerClient(SpannerSettings settings) throws IOException {
     this.settings = settings;
     this.stub = ((SpannerStubSettings) settings.getStubSettings()).createStub();
+
+    // This is dummy statement to touch the Version class that is compiled for Java 11
+    System.out.println("org.graalvm.home.Version: " + org.graalvm.home.Version.parse("1.2.3"));
```

Comment:
Test to confirm the Java 8 test fails with Java 8-incompatible class.

Reply:
The test successfully detected the incompatible class file in Java 8:

```
Error:  com.google.cloud.spanner.RetryOnInvalidatedSessionTest  Time elapsed: 0 s  <<< ERROR!
java.lang.UnsupportedClassVersionError: org/graalvm/home/Version has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

https://github.com/googleapis/java-spanner/actions/runs/5020165489/jobs/9001375691?pr=2440